interp/tests/runt.toml

```diff
@@ -161,7 +161,8 @@ fud2 --from calyx --to dat \
          --through cider \
          -s sim.data={}.data \
          -s calyx.args="--log off" \
-         {} | jq --sort-keys
+         -s jq.flags="--sort-keys" \
```

Comment:
It might be feasible to do in the data converter. Depends a little bit on how the Rust JSON library does things, since that is used to generate the JSON. I don't think it is important enough to do anything about it right now though.